host/xtest/xml/include/xml_crypto_api.h

```diff
@@ -182,37 +182,28 @@ static const uint8_t TEE_ATTR_DH_PRIVATE_VALUE_VALUE01[] = {
 	0x23, 0x7b, 0x55, 0x83
 };
 static const uint8_t TEE_ATTR_RSA_MODULUS_VALUE01[] = {
-	0xf0, 0x1a, 0x95, 0xcd, 0x5f, 0x9f, 0x1c, 0xbc, 0x5c, 0x2e, 0xc8, 0x00,
-	0x3b, 0xfa, 0xe0, 0xd5,
-	0x72, 0xea, 0xfc, 0x9e, 0x74, 0xe1, 0x02, 0x66, 0xa8, 0x13, 0x3f, 0x0c,
-	0xe6, 0x24, 0xcb, 0x1c,
-	0xa5, 0xdf, 0x64, 0xfb, 0x06, 0xd7, 0x13, 0xce, 0xaa, 0x6c, 0xee, 0x16,
-	0x7b, 0xf8, 0x92, 0xaf,
-	0xc4, 0x5b, 0x46, 0x18, 0xc6, 0x30, 0xb6, 0x04, 0x1c, 0x3a, 0x2e, 0xd7,
-	0xca, 0xb8, 0xb5, 0x00,
-	0x78, 0x89, 0xa0, 0x69, 0x37, 0x84, 0x59, 0x99, 0x0c, 0x2f, 0x00, 0xe5,
-	0x3b, 0xe1, 0x18, 0xe0,
-	0xb9, 0x2e, 0x77, 0x1d, 0x32, 0x7e, 0x5f, 0xf4, 0x18, 0xf3, 0x9f, 0x58,
-	0xc6, 0x83, 0xe2, 0x7a,
-	0xcb, 0x89, 0x18, 0xc2, 0x09, 0x84, 0x7e, 0x9d, 0x96, 0xe0, 0xb9, 0x49,
-	0x75, 0xef, 0xcf, 0xff,
-	0xf0, 0xb6, 0x18, 0xd3, 0x7a, 0xc1, 0x6f, 0x0c, 0x55, 0x33, 0xbe, 0x9d,
-	0x63, 0x06, 0xd6, 0x9f,
-	0xc1, 0xa5, 0xe9, 0xbd, 0xb1, 0xb2, 0x5d, 0x5c, 0xf9, 0xab, 0xa9, 0xb5,
-	0x6a, 0x4e, 0xa4, 0xfa,
-	0x44, 0x32, 0xd6, 0x71, 0x2e, 0x5f, 0xa6, 0x25, 0xf8, 0x40, 0x24, 0xc4,
-	0x5b, 0x61, 0x55, 0x1b,
-	0xac, 0xa3, 0x0a, 0x11, 0x8e, 0x65, 0x20, 0xda, 0x2c, 0x0d, 0xdf, 0xdb,
-	0x47, 0x6b, 0x61, 0x18,
-	0x4d, 0xfe, 0xfd, 0x2a, 0x7e, 0x77, 0x40, 0x44, 0x43, 0xc6, 0x33, 0x6c,
-	0xe5, 0x1b, 0x8d, 0x80,
-	0xf9, 0x97, 0xa2, 0xe4, 0xb9, 0x34, 0x3e, 0x28, 0x94, 0x9f, 0xbd, 0xa8,
-	0x2b, 0x0a, 0x4d, 0x1a,
-	0xa8, 0x06, 0xe5, 0x99, 0x4e, 0xb9, 0x13, 0x45, 0xc8, 0xf6, 0x0f, 0xd0,
-	0x4d, 0xbf, 0xe7, 0x8f,
-	0xed, 0xca, 0x8e, 0xf8, 0x8d, 0x87, 0x5f, 0xd4, 0xb4, 0x1a, 0x2c, 0xc9,
-	0xa7, 0x67, 0x7e, 0xb2,
-	0x1b, 0xc
```

Comment:
No need to completely change the value here. The original modulus is simply truncated, see TEE_Initial_Configuration-Test_Suite_v1_1_0_4-2014_11_07/packages/Crypto/html/TEE_Crypto_API_operationsDescriptions.html.
Remove the last `0xc` and append the following:
`0xc1, 0xce, 0xb6, 0x83, 0x7c, 0xce, 0xb4, 0x3d, 0x85, 0xc7, 0x53, 0x30, 0x7c, 0xfe, 0x85`